web_src/js/features/aria.js

```diff
@@ -85,16 +98,59 @@ function attachOneDropdownAria($dropdown) {
       if (!$item) $item = $menu.find('> .item.selected'); // when dropdown filters items by input, there is no "value", so query the "selected" item
       // if the selected item is clickable, then trigger the click event.
       // we can not click any item without check, because Fomantic code might also handle the Enter event. that would result in double click.
-      if ($item && ($item.is('a') || $item.is('.js-aria-clickable'))) $item[0].click();
+      if ($item && ($item.is('a') || $item.hasClass('js-aria-clickable'))) $item[0].click();
     }
   });
 
   // use setTimeout to run the refreshAria in next tick (to make sure the Fomantic UI code has finished its work)
-  const deferredRefreshAria = () => { setTimeout(refreshAria, 0) }; // do not return any value, jQuery has return-value related behaviors.
-  $focusable.on('focus', deferredRefreshAria);
-  $focusable.on('mouseup', deferredRefreshAria);
-  $focusable.on('blur', deferredRefreshAria);
+  // do not return any value, jQuery has return-value related behaviors.
+  const deferredRefreshAria = () => { setTimeout(refreshAria, 0) };
   $dropdown.on('keyup', (e) => { if (e.key.startsWith('Arrow')) deferredRefreshAria(); });
+
+  // if the dropdown has been opened by focus, do not trigger the next click event again.
```

Comment:
Would be better to move this code to other file, maybe `debug.js`?

Reply:
Only 2 lines are for debug: `const showDebug = false;` and `const debug =`

All other code are necessary.

Reply:
oh, ok. But still not sure I like so specific debug code being in final code.

Reply:
Why not debug to console? I don't think debug code should depend on HTML structure and generally would like it to be removed anyways. It's easy enough to manually add `console.log` to debug something, no need to litter code with it.

BTW, I'm not sure if such code would end up in the minified JS bundle. eslint may be smart enough to determine this as a static condition, but I wouldn't bet on it.

Reply:
> Why not debug to console?  It's easy enough to manually add console.log to debug something ...

Do you have some demos about your idea? I really need these logs, and I do not want to attach to a remote debugger every  time, especially when I am debugging the a11y related problems. I think these two lines also help future developers if they want to take a look at this part (confirm or debug), then they don't need to do complex setup, just change the bool from false to true.

I do not think it would affects any noticeable performance or browser behavior, so keeping them won't be a problem

Reply:
May I suggest that we use the [`debug`](https://github.com/debug-js/debug) module? E.g. start the process with `DEBUG=gitea-a11y make watch` and it should start printing to the browser console. Better than having to manually flip booleans in the code and allows debugging multiple things at once.

Oh, and please ensure that the code is compiled out in the prod build, maybe only have it when `process.env.NODE_ENV=development`.

Reply:
Could you help to provide a PR, or improve it by a separate PR later?